storage/src/vespa/storage/distributor/distributor_stripe.h

```diff
@@ -151,7 +150,7 @@ class DistributorStripe final
     }
 
     const DistributorConfiguration& getConfig() const override {
-        return _component.getTotalDistributorConfig();
+        return *_total_config;
```

Comment:
Have you verified that no of the sub components of the stripe calls _component.getTotalDistributorConfig()_?